code/modules/shuttle/special.dm

```diff
@@ -151,6 +151,10 @@
 // Bar staff, GODMODE mobs(as long as they stay in the shuttle) that just want to make sure people have drinks
 // and a good time.
 
+/mob/living/basic/drone/snowflake/bardrone/proc/round_end_depacify()
+	TRAIT_CALLBACK_REMOVE(src, TRAIT_PACIFISM, INNATE_TRAIT)
```

Comment:
```suggestion
	TRAIT_REMOVE(src, TRAIT_PACIFISM, INNATE_TRAIT)
```
You CAN use ``TRAIT_CALLBACK_REMOVE`` if you want, it would be a lot less work but you'd have no balloon alert to indicate it's gone.

Reply:
yeah sure